test/parallel/test-warn-sigprof.js

```diff
@@ -1,3 +1,4 @@
+// Flags: --inspect=0
```

Comment:
This change is because I assume handling SIGPROF should only produce the warning when the WS server is running